okex-client/src/client/mod.rs

```diff
@@ -294,6 +304,101 @@ impl OkexClient {
         }
     }
 
+    /// Open a hedging position on an instrument
+    ///
+    /// Parameters:
+    ///     inst_id(String): instrument ID e.g. "BTC-USD-SWAP" or "BTC-USD"
+    ///     trade_mode(String): "cross"
+    ///     side(String): "buy" or "sell"
+    ///     pos_side(String): "long" or "short"
+    ///     order_type(String): "market"
+    ///     size(u64): e.g. 20
+    pub async fn place_order(
```

Comment:
Need to clean up these arguments into enums instead of `String`s... can do it now or later when we have a calling code that actually uses it.

Reply:
I have created four enums: `OkexInstrumentId`, `OkexMarginMode`, `OkexOrderSide`, and `OkexPositionSide` to address this.